### PR TITLE
Post release doc update

### PR DESCRIPTION
**Pull request by pirat89**
Adding changes in docs that have been supposed to be done prior the release but have been forgotten. See the commit msgs for details. Summary:
* updated list of envars
* update reporting tutorials

### CODE REVIEW

File: docs/source/el7toel8/envars.md [hunk 1]
```diff
@@ -79,6 +80,39 @@ case they are changed. However, in some cases it's not wanted and it leads
...
+However, we are testing and discussing the possibility to switch off
+the syncrhonisation during the whole upgrade procedure in future
+-  then, in such a case it's possible someone would want to use it, but mostly
+only for the investigation purposes in case of critical failures, to see
+what last operations have been done before the failure.
+
+## LEAPP_NO_INSIGHTS_REGISTER
+
+Do not register the system into Red Hat Insights automatically, if set 1.
+It's equivalent to the `--no-rhsm-facts` leapp option.
```
Comment (matejmatuska) [inline docs/source/el7toel8/envars.md:101]:
```suggestion
It's equivalent to the `--no-insights-register` leapp option.
```

File: docs/source/el7toel8/envars.md [hunk 2]
```diff
@@ -146,3 +180,16 @@ If set to 1, leapp will disable explicit synchronization on the SQLite
 database. The positive effect is significant speed up of the leapp execution,
 however it comes at the cost of risking a corrupted database, so it is
 currently used for testing / development purposes, only.
+
+## LEAPP_DEVEL_INITRAM_NETWORK
+You can specify one of the following values: 'network-manager', 'scripts'.
+The 'scripts' value is used for a legacy dracut module when the network is not
+handled by NetworkManager.
+Using the option allows experimental upgrades, bringin up the networking inside
```
Comment (matejmatuska) [inline docs/source/el7toel8/envars.md:188]:
```suggestion
Using the option allows experimental upgrades, bringing up the networking inside
```

File: docs/source/el7toel8/envars.md [hunk 3]
```diff
@@ -15,8 +15,9 @@ Overrides the automatically detected storage device with GRUB core (e.g.
 
 ## LEAPP_NO_RHSM
 
-Do not use Red Hat Subscription Management for the upgrade. Using it has the
-same effect as using the `--no-rhsm` leapp option.
+Do not use Red Hat Subscription Management for the upgrade, if set 1.
```
Comment (matejmatuska) [inline docs/source/el7toel8/envars.md:18]:
```suggestion
Do not use Red Hat Subscription Management for the upgrade, if set to 1.
```
I would do this in the other options too

File: docs/source/el7toel8/envars.md [hunk 4]
```diff
@@ -79,6 +80,39 @@ case they are changed. However, in some cases it's not wanted and it leads
...
+
+If set to 1, leapp will explicitely synchronization on the SQLite database.
+Enabling the synchronisation has negative impact on the performance
+(sometimes very negative) however the data is more reliable in case of extreme
+situations (e.g. lost power).
+Note the synchronisation is nowadays switched off by default only during the phases
+executed before the reboot of the system to the upgrade environment which we consider
+safe, so we do not expect that anyone will want to use this option now.
+However, we are testing and discussing the possibility to switch off
+the syncrhonisation during the whole upgrade procedure in future
```
Comment (matejmatuska) [inline docs/source/el7toel8/envars.md:93]:
```suggestion
If set to 1, leapp will explicitly enable synchronization on the SQLite database.
Enabling the synchronization has negative impact on the performance
(sometimes very negative) however the data is more reliable in case of extreme
situations (e.g. lost power).
Note the synchronization is nowadays switched off by default only during the phases
executed before the reboot of the system to the upgrade environment which we consider
safe, so we do not expect that anyone will want to use this option now.
However, we are testing and discussing the possibility to switch off
the syncrhonization during the whole upgrade procedure in future
```

**Review by matejmatuska — CHANGES_REQUESTED** [review]
Found some small errors, otherwise OK

File: docs/source/el7toel8/envars.md (hunk 4 above)
Comment (matejmatuska) [reply docs/source/el7toel8/envars.md:93]: Adding fixup commit

**Comment by matejmatuska** [comment]
Added a fixup commit fixing typos and small grammar issues

File: docs/source/el7toel8/actor-rhel7-to-rhel8.md [hunk 5]
```diff
@@ -238,20 +237,26 @@ reporting.Remediation(hint='Please remove the dropped options from your scripts.
 reporting.Remediation(playbook=<link_to_playbook>)
 ```
 
-**Available tags**
+**Available Groups**
 
+Following Groups have been originally known as **Tags**:
```
Comment (mportman12) [inline docs/source/el7toel8/actor-rhel7-to-rhel8.md:242]:
```suggestion
The following groups were originally known as **Tags**:
```

File: docs/source/el7toel8/actor-rhel7-to-rhel8.md [hunk 6]
```diff
@@ -238,20 +237,26 @@ reporting.Remediation(hint='Please remove the dropped options from your scripts.
...
 ```
 'accessibility', 'authentication', 'boot', 'communication', 'drivers', 'email', 'encryption',
 'filesystem', 'firewall', 'high availability', 'kernel', 'monitoring', 'network', 'OS facts',
-'python', 'repository', 'sanity', 'security', 'selinux', 'services', 'time management',
+'post', 'python', 'repository', 'sanity', 'security', 'selinux', 'services', 'time management',
 'tools', 'upgrade process'
 ```
 
-In case of more report message tags then currently provided is needed, please open a GH issue or a PR.
+Following groups have been originally known as **Flags**:
```
Comment (mportman12) [inline docs/source/el7toel8/actor-rhel7-to-rhel8.md:250]:
```suggestion
The following groups were originally known as **Flags**:
```

File: docs/source/el7toel8/actor-rhel7-to-rhel8.md [hunk 7]
```diff
@@ -238,20 +237,26 @@ reporting.Remediation(hint='Please remove the dropped options from your scripts.
...
+```
+'failure', 'inhibitor'
+```
 
-**Flags**
+The **failure** Group is recommended to be used when the report is related to
```
Comment (mportman12) [inline docs/source/el7toel8/actor-rhel7-to-rhel8.md:255]:
```suggestion
The **failure** group is recommended when the report is related to
```

File: docs/source/el7toel8/actor-rhel7-to-rhel8.md [hunk 8]
```diff
@@ -238,20 +237,26 @@ reporting.Remediation(hint='Please remove the dropped options from your scripts.
...
+a command or other action failure.
 
-Besides the above mentioned **"inhibitor"** flag, there is also a **"failure"** flag which is recommended to use when we report a command or other action failure.
+In case of need of additional report groups, please open a GH issue or a PR,
```
Comment (mportman12) [inline docs/source/el7toel8/actor-rhel7-to-rhel8.md:258]:
```suggestion
If you need additional report groups, please open a GH issue or a PR,
```

File: docs/source/el7toel8/envars.md [hunk 9]
```diff
@@ -79,6 +80,39 @@ case they are changed. However, in some cases it's not wanted and it leads
...
+safe, so we do not expect that someone would want to use this option now.
+However, we are testing and discussing the possibility to switch off
+the synchronization during the whole upgrade procedure in the future
+-  then, in such a case it's possible someone would want to use it, but mostly
+only for the investigation purposes in case of critical failures, to see
+what last operations have been done before the failure.
+
+## LEAPP_NO_INSIGHTS_REGISTER
+
+Do not register the system into Red Hat Insights automatically, if set to 1.
```
Comment (mportman12) [inline docs/source/el7toel8/envars.md:100]:
```suggestion
If set to 1, Leapp does not register the system into Red Hat Insights automatically.
```

File: docs/source/el7toel8/envars.md [hunk 10]
```diff
@@ -79,6 +80,39 @@ case they are changed. However, in some cases it's not wanted and it leads
 in malfunction network configuration (e.g. in case the bonding is configured
 on the system). It's expected that NICs have to be handled manually if needed.
 
+## LEAPP_DATABASE_FORCE_SYNC_ON
+
+If set to 1, leapp will explicitly enable synchronization on the SQLite database.
```
Comment (mportman12) [inline docs/source/el7toel8/envars.md:85]:
```suggestion
If set to 1, Leapp will explicitly enable synchronization on the SQLite database.
```

File: docs/source/el7toel8/envars.md [hunk 11]
```diff
@@ -79,6 +80,39 @@ case they are changed. However, in some cases it's not wanted and it leads
...
+Enabling the synchronization has negative impact on the performance
+(sometimes very negative) however it is more reliable in case of extreme
+situations (e.g. lost power).
+Note the synchronization is nowadays switched off by default only during the phases
+executed before the reboot of the system to the upgrade environment which we consider
```
Comment (mportman12) [inline docs/source/el7toel8/envars.md:90]:
```suggestion
executed before the reboot of the system to the upgrade environment, which we consider
```

File: docs/source/el7toel8/envars.md [hunk 12]
```diff
@@ -79,6 +80,39 @@ case they are changed. However, in some cases it's not wanted and it leads
...
+safe, so we do not expect that someone would want to use this option now.
```
Comment (mportman12) [inline docs/source/el7toel8/envars.md:91]:
```suggestion
safe. As a result, we do not expect that someone would want to use this option now.
```

File: docs/source/el7toel8/envars.md [hunk 13]
```diff
@@ -79,6 +80,39 @@ case they are changed. However, in some cases it's not wanted and it leads
...
+However, we are testing and discussing the possibility to switch off
```
Comment (mportman12) [inline docs/source/el7toel8/envars.md:92]: Are our future plans necessary for customers to know about?

File: docs/source/el7toel8/envars.md [hunk 14]
```diff
@@ -79,6 +80,39 @@ case they are changed. However, in some cases it's not wanted and it leads
...
+only for the investigation purposes in case of critical failures, to see
+what last operations have been done before the failure.
+
+## LEAPP_NO_INSIGHTS_REGISTER
+
+Do not register the system into Red Hat Insights automatically, if set to 1.
+It's equivalent to the `--no-insights-register` leapp option.
+
+## LEAPP_NO_RHSM_FACTS
+Do not store migration information using Red Hat Subscription Manager, if set to 1.
```
Comment (mportman12) [inline docs/source/el7toel8/envars.md:104]:
```suggestion
If set to 1, Leapp does not store migration information using Red Hat Subscription Manager.
```

File: docs/source/el7toel8/envars.md [hunk 15]
```diff
@@ -79,6 +80,39 @@ case they are changed. However, in some cases it's not wanted and it leads
...
+It's equivalent to the `--no-rhsm-facts` leapp option.
+
+## LEAPP_NOGPGCHECK
+Disable RPM GPG checks (same as yum/dnf --nogpgckeck option).
```
Comment (mportman12) [inline docs/source/el7toel8/envars.md:108]:
```suggestion
Set to 1 to disable RPM GPG checks (same as yum/dnf --nogpgckeck option).
```

**Comment by mportman12** [comment]
Left some grammar-related suggestions to fix up some of the wording. Optional.

File: docs/source/el7toel8/envars.md (hunk 13 above)
Comment (pirat89) [reply docs/source/el7toel8/envars.md:92]: @mportman12  I am not looking at it as an information shared with customers, but an information shared with the community (which could include customers if they read it). From that point it should be ok to mention it. It's probably not necessary, but I felt it as a good thing to add - it's possible that same good decision could be to drop it, as it has possibly low value :D  I am still thinking whether actually to document the envar itself or drop it completely.

Comment (mportman12) [reply docs/source/el7toel8/envars.md:92]: @pirat89 Ok, in that case recommend the following rewrite. The current text is made up of very long sentences, so it's hard to follow the train of thought:
"Note the synchronization is currently disabled by default only during the phases
executed before the reboot of the system to the upgrade environment. Because this is a safe phase to disable synchronization, Red Hat does not anticipate many users will want to re-enable synchronization. 
However, Red Hat is currently testing and discussing the possibility of disabling the synchronization for all phases of the upgrade. If synchronization becomes disabled by default for the entire upgrade, users can use this environment variable for troubleshooting purposes."

Comment (pirat89) [reply docs/source/el7toel8/envars.md:92]: @mportman12  I like this, just I will exchange here Red Hat by "we" in the upstream documentation. But it sounds better now. I will update it during the start of the week. thanks!!

Comment (pirat89) [reply docs/source/el7toel8/envars.md]: dropping the part about the future plans.

**Review by MichalHe — APPROVED** [review]

**Review by fernflower — APPROVED** [review]
